src/components/SingleBubble/SingleBubble.js

```diff
@@ -28,8 +29,35 @@ class SingleBubble extends Component {
     y: 0,
   };
 
+  onBubbleClick = ev => {
+    const { onClick } = this.props;
+    if (onClick) {
+      onClick(this.props.point, ev);
+    }
+  };
+
   render() {
-    return <svg />;
+    const { onClick, height, x, width, colour, label } = this.props;
+    const d = `M${x},0a${width / 2},${height} 0 0,0 ${width},0`;
+    return (
+      <g
+        onClick={this.onBubbleClick}
+        style={{
+          cursor: onClick ? 'pointer' : 'none',
+        }}
+      >
+        <path d={d} fill={colour} />
+        <text
+          textAnchor="middle"
+          fill="white"
```

Comment:
Maybe something we could add to the props, with a default of white

Reply:
see https://github.com/digirati-co-uk/timeliner/pull/56/commits/d15eeb4407d6f0a9d082f69333b7536aa2aab94a